.github/workflows/pr_checklists.yml

```diff
@@ -0,0 +1,208 @@
+name: Pull Request Checklists
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Ah yeah that was a good call to try from a fork. For this type of things (not actually executing foreign code, however running from the context of the base repo default branch) you may need to trigger this on `pull_request_target` instead:
- https://stackoverflow.com/q/74957218/what-is-the-difference-using-pull-request-target
- https://runs-on.com/github-actions/pull-request-vs-pull-request-target/
(which then has some security considerations mentioned in the links)

That might be necessary for the `createComment`, `updateComment` APIs to actually have the token access to write to the PR anyways.